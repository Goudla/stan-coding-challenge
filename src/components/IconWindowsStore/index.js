import * as React from 'react';

// Assets
import windowsStoreSVG from './assets/windows-store.svg';

class IconWindowsStore extends React.PureComponent<*> {
  render() {
    return (
      <img src={windowsStoreSVG} alt="Windows Store" width="110" />
    );
  }
}

export default IconWindowsStore;
