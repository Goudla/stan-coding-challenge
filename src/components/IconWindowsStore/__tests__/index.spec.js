import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import IconWindowsStore from '../';

describe('<IconWindowsStore />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<IconWindowsStore />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
