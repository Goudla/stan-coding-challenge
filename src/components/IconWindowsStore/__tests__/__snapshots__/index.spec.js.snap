// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconWindowsStore /> Should render 1`] = `
<img
  alt="Windows Store"
  src="windows-store.svg"
  width="110"
/>
`;
